src/support/mtx_rw.c

```diff
@@ -433,12 +441,16 @@ __wt_writelock(WT_SESSION_IMPL *session, WT_RWLOCK *l)
 	}
 	if (set_stats) {
 		time_stop = __wt_clock(session);
+		time_diff = WT_CLOCKDIFF_US(time_stop, time_start);
 		if (F_ISSET(session, WT_SESSION_INTERNAL))
 			stats[session->stat_bucket][l->stat_int_usecs_off] +=
-			    (int64_t)WT_CLOCKDIFF_US(time_stop, time_start);
-		else
+			    (int64_t)time_diff;
+		else {
 			stats[session->stat_bucket][l->stat_app_usecs_off] +=
-			    (int64_t)WT_CLOCKDIFF_US(time_stop, time_start);
+			    (int64_t)time_diff;
```

Comment:
same as above, have we decided not to track this stat for the internal sessions?